modules/installation-creating-aws-bootstrap.adoc

```diff
@@ -213,6 +222,11 @@ parameters JSON file.
 $ aws cloudformation describe-stacks --stack-name <name>
 ----
 +
+[NOTE]
+====
+You can also view details about the stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
```

Comment:
Yes they are, but the AWS consoles provide an added dimension enabling you to see the stacks in a different format and I think holistically. When I tested the assembly I kept wondering how I can see a high level view of all of the stacks that I have created and then drill down into detail for each. The consoles provide that and my experience improved once I learned about the CloudFormation and EC2 consoles. I think the notes add value for the reader and that we should reference them in this way to at least give them a pointer to the right consoles for each operation.

Reply:
In other words, this note is meant to be about viewing the stack details in a broader sense, not as an alternative to view parameter details output in the command (although you can do that too).

Reply:
I will change this from a note to an 'Additional resources' item under this module, in order to make it clear that the consoles are an additional resource, rather than something we are suggesting only as an alternative to get the parameter details. I will do the same for the other notes like it.